### ISSUE TITLE
TIU_type package

### ISSUE THREAD

**Issue by MariellePetitDoche** [openETCS/modeling#321]
@BerndHekele:

In the definition of train_Data_T, a element is missing according Subset26 §3.18.3.2.i)
The list of national systems available on-board is necessary  to correctly proceed to the level management function, especially for the user stories proposed.

**Comment by BerndHekele** [comment]
Dear Marielle,
I will analyse the issue and change the data accordingly..

kr
Bernd

**Comment by BerndHekele** [comment]
Correction available.

**Comment by stefan-karg** [comment]
Dear Bernd, 

I adjusted the constant used for the train data at the moment according to the new data type.

Since we are simulating a ICE 3M train at the moment, I used the real data of this vehicle. I was able to insert all necessary national systems and traction systems used on the track for the scenario. 

For modeling the real ICE 3M, the fieldsize of only 3 national systems and 3 traction systems is not sufficient, since the train is equipped with more systems.

@BerndHekele
